### ISSUE TITLE
input bindings

### ISSUE THREAD

**Issue by cabmac** [stijnwop/tirePressure#9]
Hi, this is weird...since i have tire pressure in some tractors installed, the inputbindings.xml is growing and growing...the curious thing about is that the f. ex. ingame menues take a lot of time to load...when i save a game it takes 20 seconds to regress to the game...when i clear up the xml, the everything is fine like always, but then inputbindings.xml starts to grow with every minute again...

[inputbindingsxml.zip](https://github.com/stijnwop/tirePressure/files/4913547/inputbindingsxml.zip)

any idea with this problem?

**Comment by stijnwop** [comment]
Yeah, that's the whole input binding system.. can't do much about it. It can't handle mods with the same inputs for some reason
